report.py

```diff
@@ -150,10 +161,10 @@ def run_report(config, blockers, server, header, test_email, no_email, template_
 	chart_config = {
 		'type': 'pie',
 		'data': {
-			'labels': ['Success', 'Unstable', 'Failed', 'Aborted', 'Error', 'Missing'],
+			'labels': ['Success', 'Unstable', 'Failed', 'Aborted', 'Missing', 'Error'],
 			'datasets': [{
-				'backgroundColor': ['green', 'yellow', 'red', 'grey', 'brown', 'purple'],
-				'data': [num_success, num_unstable, num_failure, num_aborted, num_error, num_missing]
+				'backgroundColor': ['blue', 'yellow', 'red', 'black', 'grey', 'brown'],
```

Comment:
Isn't Aborted color grey in jenkins?

Reply:
Nope, "Aborted" is black and "Not built" is gray

Reply:
ah yes, maybe it's changed with the new jenkins CI